docs/src/opt_cep.md

```diff
@@ -55,25 +66,35 @@ An overview of the variables used in the CEP is provided in the table:
 
-| description                                                                          |  unit            | configuration           | values                                      | type           | default value |
+| description                                        |  unit            | configuration           | values                                      | type           | default value |
 |--------------------------------------------------------------------------------------|------------------|-------------------------|---------------------------------------------|----------------|---------------|
-| enforce a CO2-limit                                                                  | kg-CO2e/MWh    | `co2_limit`               | >0                                          | ::Number       | Inf           |
-| including existing infrastructure (no extra costs)                                   | -                | `existing_infrastructure` | true or false                               | ::Bool         | false         |
-| type of storage implementation                                                       | -                | `storage`                 | "none", "simple" or "seasonal"              | ::String       | "none"        |
-| allowing transmission                                                                | -                | `transmission`            | true or false                               | ::Bool         | false         |
-| fix. var and CEO to dispatch problem | -                | `fixed_design_variables`  | design variables from design run or nothing | ::OptVariables | nothing       |
-| allowing lost load (necessary for dispatch)                        | price/MWh        | `lost_el_load_cost`       | >1e6                                        | ::Number       | Inf           |
-| allowing lost emission (necessary for dispatch)                    | price/kg-CO2e | `lost_CO2_emission_cost`  | >700                                        | ::Number       | Inf           |
+| enforce an emission-limit                          | kg-impact/MWh-carrier | `limit_emission`               | Dict{String,Number}(impact/carrier=>value)                                      | ::Dict{String,Number}       | Dict{String,Number}()           |
+| including existing infrastructure (no extra costs) and limit infrastructure   | -                | `infrastructure`| Dict{String,Array}("existing"=>[tech-groups...], "limit"=>[tech-groups...])                             | ::Dict{String,Array}       | Dict{String,Array}("existing"=>["demand"])         |
+| type of storage implementation                     | -                | `storage_type`                 | "none", "simple" or "seasonal"              | ::String       | "none"        |
+| allowing conversion (necessary for storage)        | -                | `conversion`            | `true` or `false`                               | ::Bool         | false         |
```

Comment:
What is conversion and why is it necessary? If we can avoid having settings that depend on each other, that would be great for the new user (e.g. storage and conversion).